doc/sphinx/addendum/universe-polymorphism.rst

```diff
@@ -183,6 +180,9 @@ Cumulative, NonCumulative
    are convertible based on the universe variances; they do not need
    to be equal.
 
+   This means that two instances of the same inductive type (family)
+   are convertible only if all the universes are equal.
+
    .. exn:: The cumulative and noncumulative attributes can only be used in a polymorphic context.
```

Comment:
This error message should be updated to reflect the new syntax. Proposal: "The universes(cumulative) attribute can only be used in a polymorphic context.

Reply:
Updated, thanks @zimmi48 !